platform-common/src/test/java/com/softicar/platform/common/ui/image/ImagesTest.java

```diff
@@ -62,6 +63,35 @@ public void testCountPixelsWithColor() {
 		assertEquals(3067, Images.countPixelsWithColor(image, textColor));
 	}
 
+	@Test
+	public void testIsSupportedWithMimeTypeJpg() {
```

Comment:
why is only `Jpg` in the test method name?

Reply:
Oops. Artifact.